.docker/Dockerfile

```diff
@@ -64,12 +64,19 @@ RUN mkdir /library_pkgs && \
     mv "/scanCONTROL-Linux-SDK/libmescan/builddir/mescan_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs && \
     mv "/scanCONTROL-Linux-SDK/libllt/builddir/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs
 
-FROM ros:noetic-ros-core
+FROM osrf/ros:noetic-desktop-full
```

Comment:
The full desktop image is overkill. Please use something like `ros:noetic-ros-base` as those include the essential build tools already.